test/e2e_node/services/services.go

```diff
@@ -65,14 +65,16 @@ func NewE2EServices(monitorParent bool) *E2EServices {
 func (e *E2EServices) Start() error {
 	var err error
 	if !framework.TestContext.NodeConformance {
+		if e.services, err = e.startInternalServices(); err != nil {
+			return fmt.Errorf("failed to start internal services: %v", err)
+		}
 		// Start kubelet
 		e.kubelet, err = e.startKubelet()
 		if err != nil {
 			return fmt.Errorf("failed to start kubelet: %v", err)
 		}
 	}
-	e.services, err = e.startInternalServices()
-	return err
+	return nil
```

Comment:
I think our problem for #95877, we don't even start the services in conformance mode now.

Reply:
Perhaps this was moved because we start the kubelet [here in conformance tests?](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/remote/node_conformance.go#L176) So I guess we need to separate starting the internal services.

Reply:
Ya, that's why the kubelet start is inside.

But there isn't any change to node_conformance to start the services.